tools/accuracy_checker/configs/README.md

```diff
@@ -0,0 +1,34 @@
+# How to use configuration files
+
+## Structure
+
+Configuration file declares validation process. Every model has to have entry in `models` list. Each entry has to contain distinct `name`, `launchers` and `datasets` sections.
+
+Example:
+
+```yaml
+models:
+  - name: model_name
+
+    launchers:
+      - framework: dlsdk
+        adapter: adapter_name
+
+    datasets:
+      - name: dataset_name
+```
+
+## Location
+
+Configuration file `accuracy-check.yml` is located in the `models` folder with model to which it belongs.
```

Comment:
```suggestion
Predifined configuration file `accuracy-check.yml` for each Open Model Zoo model can be found in the model directory
```
it will be nice if you provide example